### PR TITLE
Added test case related to WPE-469

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Added below files as a part of WPE-469 :

WebKit_Startup_002.test.js

### CODE REVIEW

File: src/webkit/WebKit_Startup_002.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  title: 'WPEWebkit shutdown of Framework robustness test',
+  description: 'Starts WPEWebkit and stops Framework. Checks if everything is shutdown correctly',
+  repeat: 30,
+  setup() {
+    return this.$sequence([
+      () => pluginDeactivate.call(this, constants.youTubePlugin),
+      () => pluginDeactivate.call(this, constants.netFlixPlugin),
+    ])
+  },
+  teardown: restartFramework,
```
Comment (wouterlucas) [inline src/webkit/WebKit_Startup_002.test.js]: No need

File: src/webkit/WebKit_Startup_002.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  steps: [
+    {
+      description: 'Activate WebKit Plugin and check whether it is resumed correctly',
+      test: webKitBrowserOps,
```
Comment (wouterlucas) [inline src/webkit/WebKit_Startup_002.test.js]: Already commented on other PR as well, this should be renamed to something more concrete. Like webKitBrowserRestart or webKitBrowserRestartAndResume.

File: src/webkit/WebKit_Startup_002.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      description: 'Check if WebKitImplementation rpcprocess is gone',
+      test: checkIfProcessIsRunning,
+      params: constants.webKitImplementation,
+      assert: false,
+    },
+    {
+      description: 'Start Framework',
+      test: startFramework,
+      assert: true,
+    },
```
Comment (wouterlucas) [inline src/webkit/WebKit_Startup_002.test.js]: I know this is unreleated to the test case convert, but looking at it we likely want to add something to check if WPEFramework came up successfully. Like a few calls to the controller to valdiate it is behaving as expected before we rerun the test.

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/webkit/WebKit_Startup_002.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/webkit/WebKit_Startup_002.test.js]: Removed restartFramework

File: src/webkit/WebKit_Startup_002.test.js (hunk 3 above)
Comment (sandeep-vedam) [reply src/webkit/WebKit_Startup_002.test.js]: Added validation to check whether the controller is responding and the response from controller is object or not.

File: src/webkit/WebKit_Startup_002.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/webkit/WebKit_Startup_002.test.js]: Renamed method name to a better one.

**Pull request merged**